frontend/src/components/LandingPage/Tools/Tools.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import { Card, CardBody, CardLink, CardHeader, CardText } from 'reactstrap';
+import './Tools.css'
+
+function Tools() 
+{
+    return (
+    <>
+    <h1>Tools We Used</h1>
+    <ToolCard
+        title="Reactstrap"
+        link="https://reactstrap.github.io/"
+        description="We use Reactstrap to allow for simple Bootstrap component creations."
+    />
+    <ToolCard
+        title="Jira"
+        link="https://www.atlassian.com/software/jira"
+        description="We use Jira as our project management software to track issues and organize sprints for our development"
+    />
+    <ToolCard
+        title="Uber API"
+        link="https://developer.uber.com/"
+        description="The Uber API is used to give users the option of easily booking a ride to their destination."
+    />
+    <ToolCard
+        title="Yelp API"
+        link="https://www.yelp.com/developers"
+        description="The Yelp API gives users easy access to reviews and feedback on potential destinations."
+    />
+    </>
+    )
+}
+
+const ToolCard = (props) => {
```

Comment:
Although I completely understand what you are doing here, can we keep the files to one component each.

Reply:
should i move the component to a new file or just get rid of it?

Reply:
Whichever of the two you would like. If you choose to keep it, just make sure to put it in another file and import it.